spec/requests/api/v1/items_request_spec.rb

```diff
@@ -36,4 +36,89 @@
 
     expect(item[:data][:attributes]).to have_key(:name)
   end
+
+  it "can get one item by id" do
+    id =  create(:item).id
+
+    get "/api/v1/items/#{id}"
+
+    expect(response).to be_successful
+
+    item = JSON.parse(response.body, symbolize_names: true)
+
+    expect(item[:data][:id]).to eq("#{id}")
+
+    expect(item[:data]).to have_key(:id)
+    expect(item[:data]).to have_key(:type)
+    expect(item[:data]).to have_key(:attributes)
+
+    expect(item[:data][:attributes]).to have_key(:name)
+  end
+
+  it "can create a new item" do
+    merchant_id = create(:merchant).id
+
+    headers = { "CONTENT_TYPE" => "application/json"}
+    post "/api/v1/items", :params => {
```

Comment:
Passing params like this made it much easier to read in the test, and much easier to figure out how to read the params in the controller.